packages/docs-app/src/examples/core-examples/collapsibleListExample.tsx

```diff
@@ -72,6 +72,7 @@ export class CollapsibleListExample extends BaseExample<ICollapsibleListExampleS
                 <RadioGroup
                     key="collapseFrom"
                     name="collapseFrom"
+                    inline={true}
```

Comment:
what's this?

Reply:
the _other_ inline modifier, the one that makes checkboxes etc `display: inline-block` so they'll lie on the same line